priv/www/js/global.js

```diff
@@ -425,9 +425,15 @@ var HELP = {
 
     'filter-regex' :
     'Whether to enable regular expression matching. Both string literals \
-    and regular expressions are matched in a case-insensitive manner.<br/></br/> \
+    and regular expressions are matched in a case-insensitive manner.<br/><br/> \
     (<a href="https://developer.mozilla.org/en/docs/Web/JavaScript/Guide/Regular_Expressions" target="_blank">Regular expression reference</a>)',
 
+    'consumer-active' :
+    'Whether the consumer is active or not. Consumers are active by default. For a quorum queue, a consumer can be inactive \
```

Comment:
Can we first mention single-active consumer scenario and then quorum queues?

Reply:
It would be nice to have some explanation of what does the word "active" mean. Something like "can get messages from the queue".